browser/resources/settings/brave_settings_overrides.js

```diff
@@ -325,12 +327,26 @@ BravePatching.RegisterPolymerTemplateModifications({
     }
   },
   'settings-people-page': (templateContent) => {
-    // Import item needs to know it's the first in the section
-    const importItem = templateContent.querySelector('#importDataDialogTrigger[on-click="onImportDataTap_"]')
-    if (!importItem) {
-      console.error('[Brave Settings Overrides] Could not find import item in people_page')
+    // People page needs to think it's in the getStarted section, since it is
+    // (we remove the People section as a separate section).
+    const page = templateContent.querySelector('settings-animated-pages[section=people]')
+    page.setAttribute('section', 'getStarted')
+    const profileTemplate = templateContent.querySelector('template[is="dom-if"][if="[[diceEnabled_]]"]')
```

Comment:
this is ugly as is subject to change in chromium and not named anything nice. However, when it does change soon, it will be to remove the `diceEnabled` flag, and hopefully remove this _'if'_ template entirely, making a cleaner selector possible.